x-pack/test/fleet_api_integration/apis/uninstall_token/get.ts

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import {
+  AGENT_POLICY_API_ROUTES,
+  CreateAgentPolicyResponse,
+  UNINSTALL_TOKENS_SAVED_OBJECT_TYPE,
+} from '@kbn/fleet-plugin/common';
+import { UNINSTALL_TOKEN_ROUTES } from '@kbn/fleet-plugin/common/constants';
+import { GetUninstallTokensResponse } from '@kbn/fleet-plugin/common/types/rest_spec/uninstall_token';
+import * as uuid from 'uuid';
+import { testUsers } from '../test_users';
+import { FtrProviderContext } from '../../../api_integration/ftr_provider_context';
+
+export default function (providerContext: FtrProviderContext) {
+  const { getService } = providerContext;
+  const supertest = getService('supertest');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('Uninstall Token API', () => {
+    let policyIds: string[];
+
+    before(async () => {
+      await cleanSavedObjects();
+    });
+
+    after(async () => {
+      await cleanSavedObjects();
+    });
+
+    describe('pagination', () => {
+      before(async () => {
+        policyIds = await generatePolicies(20);
+      });
+
+      after(async () => {
+        await cleanSavedObjects();
+      });
+
+      it('should return tokens for all policies if number is below default perPage', async () => {
```

Comment:
Should this say `should return tokens for all policies if number of policies is below default perPage`?

Reply:
64a9118